### DISCUSSION TITLE
Utilise standard ruleset for editor

### DISCUSSION THREAD

**Discussion by TheDanielDoyle** [alex-hedley/BlazorInteractive#3] [Ideas]
We can use editorconfig files to standardise our code syntax.

The following Blazor site has a wizard to generate the file.

[https://kristofferstrube.github.io/EditorConfigWizard/wizard](https://kristofferstrube.github.io/EditorConfigWizard/wizard)

**Comment by AlexHedley** [comment]
https://github.com/alex-hedley/BlazorInteractive/commit/1c7b894f4f8458e937575769a6d88c5c54dff8de
